package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "lens-protocol",
+  "name": "@lens-protocol/contracts",
```

Comment:
Idk if it makes sense to change the repository name here, I think `lens-protocol` is already self-explanatory, wdyt?

Reply:
As far as I understand this is the name that the npm package will have after creating it. So to be able to do `npm install @lens-protocol/contracts` we need to change it here. Basically setting `@org-name/package-name`

Reply:
But the organization should be Aave no?

So something like aave/lens-protocol, is that how we do it for V3?

Reply:
As example we can take Aave V3, here the [package.json](https://github.com/aave/aave-v3-core/blob/master/package.json#L2) and the [npm package](https://www.npmjs.com/package/@aave/core-v3)

Reply:
Could be `@aave/lens-protocol` or `@lens-protocol/contracts`. We have both orgs, I understood we wanted to go over `@lens-protocol` org, but let's double-check with the rest of the team.

Reply:
Ok, let's go with `@aave/lens-protocol` to match the repo